Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -475,7 +481,10 @@ namespace Multiplayer
     void MultiplayerEditorSystemComponent::Connect()
     {
         ++m_connectionAttempts;
-        AZ_Printf("MultiplayerEditor", "Editor tcp connection attempt #%i.", m_connectionAttempts)
+
+        AZStd::string message = AZStd::string::format("Editor tcp connection attempt #%i.", m_connectionAttempts);
```

Comment:
You could also use `azsnprintf` with a char buffer to avoid heap allocations for status messages over 10 characters or whatever our limit is for on-the-stack string optimization.

Reply:
Fixed! nice, azsnprintf will also compile time catch if you're n length is too big for the array.